docs/jitserver.md

```diff
@@ -62,16 +65,36 @@ You can use command line options to further configure the JITServer and the clie
 - [`-XX:[+|-]JITServerShareROMClasses`](xxjitservershareromclasses.md): Specifies whether the server shares cached ROM classes between clients
 - [`-XX:[+|-]JITServerLocalSyncCompiles`](xxjitserverlocalsynccompiles.md): Improves performance for real-time applications by compiling synchronous JIT compilations locally, with a remote asynchronous recompilation scheduled at a later point
 - [`-XX:[+|-]JITServerLogConnections`](xxjitserverlogconnections.md): Enables logging of connection/disconnection events between the server and the client
+- [`-XX:JITServerAOTCacheName`](xxjitserveraotcachename.md): Specifies the name of the server-side AOT cache to use
```

Comment:
To follow the style above, this should be:
"[`-XX:JITServerAOTCacheName=<name>`](xxjitserveraotcachename.md): Specifies the name of the server-side AOT cache to use"

Reply:
I am thinking that we should also list the two new options for JITServer metrics.